### PR TITLE
refactor: vcf2cytosure container

### PR DESCRIPTION
**Pull request by khurrammaqbool**
### This PR:

Changed: The base of the container.
Fixed: dependency versions for the installation of vcf2cytosure.

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Review by ivadym — COMMENTED** [review]
I ran the container build again on your PR and it seems to have resolved itself:

https://github.com/Clinical-Genomics/BALSAMIC/actions/runs/5001681295/jobs/9044597000?pr=1126

**Comment by khurrammaqbool** [comment]
> I ran the container build again on your PR and it seems to have resolved itself:
> 
> https://github.com/Clinical-Genomics/BALSAMIC/actions/runs/5001681295/jobs/9044597000?pr=1126


This pull request fixes the versions of the dependencies as we have started to do to build all the containers. It is similar to what is done in https://github.com/Clinical-Genomics/BALSAMIC/pull/1096. Additionally, I have removed conda, which will make this container more stable, reduce the size and little bit faster to build.

File: BALSAMIC/containers/vcf2cytosure/vcf2cytosure.yaml [hunk 1]
```diff
@@ -1 +1 @@
-- vcf2cytosure=0.8.1
+- vcf2cytosure=0.8
```
Comment (ivadym) [inline BALSAMIC/containers/vcf2cytosure/vcf2cytosure.yaml:1]: The [version is 0.8.1 ](https://github.com/NBISweden/vcf2cytosure/blob/master/__version__.py)actually, the only thing it has been not bumped... 
```suggestion
- vcf2cytosure=0.8.1
```

File: BALSAMIC/containers/vcf2cytosure/Dockerfile [hunk 2]
```diff
@@ -1,28 +1,54 @@
...
-LABEL base.image="continuumio/miniconda3:4.10.3-alpine"
+ARG CONTAINER_NAME
+ENV WORK_DIR /opt/
+ENV VENV ${WORK_DIR}/venv
+
+RUN apt-get update && \
+     apt-get -y upgrade && \
+     apt-get -y install -y --no-install-recommends git tabix && \
+     apt-get clean && \
+     rm -rf /var/lib/apt/lists/*
```
Comment (ivadym) [inline BALSAMIC/containers/vcf2cytosure/Dockerfile:11]: This is my OCD talking, but let's be consistent with the tabs: `tabstop=4`  (and same thing below)
```suggestion
RUN apt-get update && \
    apt-get -y upgrade && \
    apt-get -y install -y --no-install-recommends git tabix && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```

File: BALSAMIC/containers/vcf2cytosure/Dockerfile [hunk 3]
```diff
@@ -1,28 +1,54 @@
...
+     apt-get -y install -y --no-install-recommends tabix && \
+     apt-get clean && \
+     rm -rf /var/lib/apt/lists/*
 
-COPY . /${WORK_DIR}
+ENV WORK_DIR /opt/
+ENV VENV ${WORK_DIR}/venv
 
-RUN apk add --no-cache bash gcc git python3 gzip
+COPY --from=builder /opt /opt
```
Comment (ivadym) [inline BALSAMIC/containers/vcf2cytosure/Dockerfile:45]:
```suggestion
COPY --from=builder ${WORK_DIR} ${WORK_DIR}
```

File: BALSAMIC/containers/vcf2cytosure/Dockerfile [hunk 4]
```diff
@@ -1,28 +1,54 @@
...
-    && git checkout v0.8 \
-    && pip install --no-cache-dir .
+ENV PATH="${VENV}/bin:$PATH"
 
-RUN cd /${WORK_DIR}/BALSAMIC/containers/${CONTAINER_NAME}/ && /bin/sh ${CONTAINER_NAME}.sh ${CONTAINER_NAME}
+RUN adduser --disabled-password --gecos '' ubuntu &&\
+chsh -s /bin/bash && mkdir -p /home/ubuntu
 
-RUN rm -rf /${WORK_DIR:?} && conda clean --all --yes
+USER    ubuntu
```
Comment (ivadym) [inline BALSAMIC/containers/vcf2cytosure/Dockerfile:52]:
```suggestion
USER ubuntu
```

File: BALSAMIC/containers/vcf2cytosure/Dockerfile [hunk 5]
```diff
@@ -1,28 +1,54 @@
...
-RUN cd /opt \
-    && git clone https://github.com/NBISweden/vcf2cytosure.git \
-    && cd /opt/${CONTAINER_NAME}/ \
-    && git checkout v0.8 \
-    && pip install --no-cache-dir .
+ENV PATH="${VENV}/bin:$PATH"
 
-RUN cd /${WORK_DIR}/BALSAMIC/containers/${CONTAINER_NAME}/ && /bin/sh ${CONTAINER_NAME}.sh ${CONTAINER_NAME}
+RUN adduser --disabled-password --gecos '' ubuntu &&\
+chsh -s /bin/bash && mkdir -p /home/ubuntu
```
Comment (ivadym) [inline BALSAMIC/containers/vcf2cytosure/Dockerfile:50]:
```suggestion
RUN adduser --disabled-password --gecos '' ubuntu && \
    chsh -s /bin/bash && \
    mkdir -p /home/ubuntu
```

**Review by ivadym — APPROVED** [review]
Well done ⭐ Nothing major to add, just some little suggestions 👍

**Comment by khurrammaqbool** [comment]
Thanks you 🙏  @ivadym for the suggestions.

**Pull request merged**